generic_config_updater/generic_config_updater.conf.json

```diff
@@ -39,6 +39,9 @@
         },
         "DHCP_SERVER": {
             "services_to_validate": [ "dhcp-relay" ]
+        },
+        "VLAN": {
+            "services_to_validate": [ "vlan-service" ]
```

Comment:
>services_to_validate [](http://example.com/codeflow?start=13&length=20)

According to the design doc [https://github.com/Azure/SONiC/blob/master/doc/config-generic-update-rollback/Json_Change_Application_Design.md#3212-metadata](https://github.com/Azure/SONiC/blob/master/doc/config-generic-update-rollback/Json_Change_Application_Design.md#3212-metadata). It fits better as `restart-command` than `services_to_validate`

Reply:
This is not blindly restart. 
For example, in case of VLAN table update, we call vlan_validator, which *may* or may not restart dhcp_sevice.
Again a validator, could potentially restart more than one and possibly none too, as in vlan case.

So I prefer this name validator, which is table specific.

Reply:
I see the intelligence of restarting dhcp_service for vlan-service. However, the validator is design as read-only operations for the purpose of validation, including validating services are completely restarted and healthy.

Reply:
I don't get your last comments. But Design doc updated.